core/worker.go

```diff
@@ -742,6 +756,9 @@ func (w *worker) GeneratePendingHeader(block *types.WorkObject, fill bool, txs t
 	w.printPendingHeaderInfo(work, newWo, start)
 	work.utxosCreate = nil
 	work.utxosDelete = nil
+	work.coinbaseLockupsCreated = nil
```

Comment:
sould these be nil or empty?

Reply:
It's just a memory release at the end of Generate, it's not used. Might help the garbage collector, might not